test/lib/widgets/product_license_translations_test.rb

```diff
@@ -17,85 +17,74 @@
 require "y2packager/widgets/product_license_translations"
 require "y2packager/product"
 
+RSpec::Matchers.define :array_not_including do |x|
```

Comment:
Can this new rspec matcher leak to the other tests executed *after* this one? (When using `rake test:unit`.) Would it be then better to share it explicitly in the `test_helper.rb`?

Reply:
Yes, I agree sharing it explicitly sharing it in the test helper. Just I forget to do it because I wanted to check first if it is really necessary o there is another approach.